src/uct/ib/rc/base/rc_ep.h

```diff
@@ -109,6 +109,15 @@ enum {
     UCT_RC_CHECK_CQE_RET(_iface, _ep, UCS_ERR_NO_RESOURCE) \
     UCT_RC_CHECK_TXQP_RET(_iface, _ep, UCS_ERR_NO_RESOURCE)
 
+/**
+ * All RMA and AMO operations are not allowed if no RDMA_READ credits.
+ * Otherwise operations ordering can be broken (which fence operation
+ * relies on).
+ */
+#define UCT_RC_CHECK_RMA_RES(_iface, _ep, _txqp) \
```

Comment:
[minor][q] maybe better to get `txqp` from `_ep`? wdyt?

Reply:
i'm going to use it with dc, where txqp is part of iface

Reply:
> i'm going to use it with dc, where txqp is part of iface

thank you for the clarification!